ckanext/canada/templates/public/snippets/sort_by.html

```diff
@@ -11,14 +11,15 @@
 <div class="form-group">
   <label for="field-order-by" class="control-label">{{ _('Order by') }}</label>
   <select id="field-order-by" name="sort" class="form-control input-sm" onchange="this.form.submit()">
+    <option value="score desc,metadata_modified desc" >{{ _('Relevance') }}</option>
```

Comment:
is there a reason you use `value="score desc,metadata_modified desc"` instead of `value=""` here?